getting-started/templates/systemlink-values.yaml

```diff
@@ -617,27 +617,51 @@ sl-jupyterhub:
     annotations:
       nginx.ingress.kubernetes.io/proxy-body-size: 100m
 
-## Configuration for Notebook Execution workers
+## Configuration for Argo Workflows.
 ##
-nbexecworker:
-  ## Customize routes to back-end APIs
-  ##
-  apiRoutes:
-    ## Sets the release name prefix for the Jupyter Enterprise Gateway service running on the cluster.
-    # <ATTENTION> This must be set to the release name used to install the ni-helm/systemlinkadmin chart.
-    ##
-    jupyterEnterpriseGatewayReleaseName: "systemlink-admin"
-    ## Sets the service name for the Jupyter Enterprise Gateway instance running on the cluster.
-    ##
-    jupyterEnterpriseGatewayServiceName: "enterprise-gateway"
-    ## Sets the namespace name for the Jupyter Enterprise Gateway service running on the cluster.
-    # <ATTENTION> This must be set to the namespace where the ni-helm/systemlinkadmin chart was installed.
-    ##
-    jupyterEnterpriseGatewayNamespace: "systemlink-admin"
-    ## Sets the port number of for the Jupyter Enterprise Gateway service running on the cluster.
-    ##
-    jupyterEnterpriseGatewayPort: "8888"
-    ## If set, overrides the other jupyterEnterpriseGateway options and configures the service URL directly.
+argoworkflows:
+  argo-workflows:
+    images:
+      pullSecrets: *pullSecrets
```

Comment:
This should also be recorded as a breaking change in the breaking change document. GM will need to pull this change into their local configuration.